evap/evaluation/tests/test_models.py

```diff
@@ -657,6 +657,28 @@ def test_missing_email_address():
         template = EmailTemplate.objects.get(name=EmailTemplate.STUDENT_REMINDER)
         template.send_to_user(user, {}, {}, False, None)
 
+    def test_send_multi_alternatives_email(self):
+        template = EmailTemplate(subject='Example', plain_content='Example plain content', html_content='<p>Example html content</p>')
+        template.send_to_user(self.user, {}, {}, False)
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertTrue(isinstance(mail.outbox[0], mail.message.EmailMultiAlternatives))
+        self.assertEqual(mail.outbox[0].body, 'Example plain content')
+        self.assertEqual(len(mail.outbox[0].alternatives), 1)
+        self.assertEqual(mail.outbox[0].alternatives[0][1], 'text/html')
+        self.assertIn('<p>Example html content</p>', mail.outbox[0].alternatives[0][0])
+
+    def test_copy_plain_content_on_empty_html_content(self):
```

Comment:
Maybe add a newline to check that it also is converted to a HTML linebreak?

Reply:
Yep, fixed in `27d8262`. (I added this case to `test_copy_plain_content_on_empty_html_content` because it fits well in there)